### PR TITLE
Add Self Signed certificate to motherbot

### PR DESCRIPTION
**Pull request by angie4u**

### CODE REVIEW

File: aks/charts/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+secret.yaml
```
Comment (h2floh) [inline aks/charts/.gitignore:1]: Good security practice

File: samples/VAonAKS/MotherBot/MotherBot/Program.cs [hunk 2]
```diff
@@ -26,9 +26,9 @@ public static void Main(string[] args)
                 //// if not standard value of 'ConnectionStrings:AppConfig' will be used
                 .ConfigurationFromAzureAppConfig()
 
-                //// The KeyVault and Certificate Name can be specified as parameters or in ENV 'KeyVaultName' and ENV 'CertificateName'
-                //// For this example it is not needed (no SSL calls to befriended microservices)
-                ////.ConfigureRootCAFromKeyVault()
+                // The KeyVault and Certificate Name can be specified as parameters or in ENV 'KeyVaultName' and ENV 'CertificateName'
+                // For this example it is not needed (no SSL calls to befriended microservices)
+                .ConfigureRootCAFromKeyVault("VmAMaster0711Kv","LocalhostK8s")
```
Comment (h2floh) [inline samples/VAonAKS/MotherBot/MotherBot/Program.cs]: Should be changed to other ENV variables for better reuse

**Review by h2floh — CHANGES_REQUESTED** [review]
Please change hard coded values to ENV variables.

File: samples/VAonAKS/MotherBot/MotherBot/Program.cs (hunk 2 above)
Comment (angie4u) [reply samples/VAonAKS/MotherBot/MotherBot/Program.cs]: I changed those setting to use ENV variables. Please review my change! 💃

**Review by h2floh — APPROVED** [review]
Thanks

**Pull request merged**